Build is broken

The build for this repo appears to be broken:

```
experimental/elasticloadbalancingv2/targetGroup.ts:60:29 - error TS2532: Object is possibly 'undefined'.
60             containerPort: +port,
                               ~~~~
experimental/elasticloadbalancingv2/targetGroup.ts:65:9 - error TS2322: Type 'Output<{ containerPort: Output<number | undefined>; targetGroupArn: Output<string>; }[]>' is not assignable to type 'Input<Input<ContainerLoadBalancer>[]>'.
  Type 'Output<{ containerPort: Output<number | undefined>; targetGroupArn: Output<string>; }[]>' is not assignable to type 'Output<Input<ContainerLoadBalancer>[]>'.
    Type '{ containerPort: Output<number | undefined>; targetGroupArn: Output<string>; }[]' is not assignable to type 'Input<ContainerLoadBalancer>[]'.
      Type '{ containerPort: Output<number | undefined>; targetGroupArn: Output<string>; }' is not assignable to type 'Input<ContainerLoadBalancer>'.
        Type '{ containerPort: Output<number | undefined>; targetGroupArn: Output<string>; }' is not assignable to type 'ContainerLoadBalancer'.
          Types of property 'containerPort' are incompatible.
            Type 'Output<number | undefined>' is not assignable to type 'Input<number>'.
              Type 'Output<number | undefined>' is not assignable to type 'Output<number>'.
                Type 'number | undefined' is not assignable to type 'number'.
                  Type 'undefined' is not assignable to type 'number'.
65         return this.dependencies().apply(_ => [{
           ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
66             containerPort: this.instance.port,
   ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
67             targetGroupArn: this.instance.arn,
   ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
68         }]);
   ~~~~~~~~~~~~
Found 2 errors.
```

--- a/nodejs/aws-infra/examples/ec2/package.json
+++ b/nodejs/aws-infra/examples/ec2/package.json
@@ -8,8 +8,8 @@
         "build": "tsc"
     },
     "dependencies": {
-        "@pulumi/pulumi": "dev",
-        "@pulumi/aws": "dev",
+        "@pulumi/pulumi": "latest",
+        "@pulumi/aws": "latest",
         "node-fetch": "^1.7.3",
         "redis": "^2.8.0"
     },

--- a/nodejs/aws-infra/examples/fargate/package.json
+++ b/nodejs/aws-infra/examples/fargate/package.json
@@ -8,8 +8,8 @@
         "build": "tsc"
     },
     "dependencies": {
-        "@pulumi/pulumi": "dev",
-        "@pulumi/aws": "dev",
+        "@pulumi/pulumi": "latest",
+        "@pulumi/aws": "latest",
         "node-fetch": "^1.7.3",
         "redis": "^2.8.0"
     },

--- a/nodejs/aws-infra/experimental/elasticloadbalancingv2/targetGroup.ts
+++ b/nodejs/aws-infra/experimental/elasticloadbalancingv2/targetGroup.ts
@@ -57,13 +57,13 @@ export abstract class TargetGroup
 
     public containerPortMappings(): pulumi.Input<pulumi.Input<aws.ecs.PortMapping>[]> {
         return pulumi.output([this.instance.port, this.dependencies()]).apply(([port]) => [{
-            containerPort: +port,
+            containerPort: +port!,
         }]);
     }
 
     public containerLoadBalancers(): pulumi.Input<pulumi.Input<x.ecs.ContainerLoadBalancer>[]> {
         return this.dependencies().apply(_ => [{
-            containerPort: this.instance.port,
+            containerPort: this.instance.port.apply(p => p!),
             targetGroupArn: this.instance.arn,
         }]);
     }

--- a/nodejs/aws-infra/package.json
+++ b/nodejs/aws-infra/package.json
@@ -11,8 +11,8 @@
     "homepage": "https://pulumi.io",
     "repository": "https://github.com/pulumi/pulumi-aws-infra",
     "dependencies": {
-        "@pulumi/pulumi": "dev",
-        "@pulumi/aws": "dev",
+        "@pulumi/pulumi": "^0.16.9",
+        "@pulumi/aws": "^0.16.5",
         "@pulumi/docker": "^0.16.1"
     },
     "devDependencies": {

